harness/determined/keras/_tf_keras_trial.py

```diff
@@ -485,11 +485,11 @@ def run(self) -> None:
                 break
 
             elif wkld.kind == workload.Workload.Kind.COMPUTE_VALIDATION_METRICS:
-                response = {
-                    "metrics": self.compute_validation_metrics(),
-                    "stop_requested": self.context.get_stop_requested(),
-                }
-                response_func(response)
+                response_func(
+                    det.util.wrap_metrics(
```

Comment:
nit: `util.wrap_metrics(`

Reply:
gonna hold off --  we already use `det.util` in other places 👀